cmd/genji/shell/command.go

```diff
@@ -315,22 +314,22 @@ func runDumpCmd(db *genji.DB, tables []string, w io.Writer) error {
 	}
 
 	for i, table := range tables {
-		err = dumpTable(tx, table, w)
-		if err != nil {
+		// Blank separation between tables.
+		if i > 0 {
+			if _, err := fmt.Fprintln(w, ""); err != nil {
+				return err
+			}
+		}
+
+		if err = dumpTable(tx, table, w); err != nil {
 			// If table doesn’t exist we skip it.
 			if errors.Is(err, database.ErrTableNotFound) {
 				continue
 			}
-			_, err = fmt.Fprintln(w, "COMMIT;")
+			_, err = fmt.Fprintln(w, "ROLLBACK;")
```

Comment:
I am not sure to understand this changes here. Could you elaborate please?

Reply:
If an error occurs during the dump we should `ROLLBACK`, it's a correction.

Reply:
Oops, got it!